ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala

```diff
@@ -576,16 +571,9 @@ object CliConfig {
       opt[Long]("max-lf-value-translation-cache-entries")
         .optional()
         .text(
-          s"The maximum size of the cache used to deserialize Daml-LF values, in number of allowed entries. By default, nothing is cached."
-        )
-        .action((maximumLfValueTranslationCacheEntries, config) =>
-          config.copy(
-            lfValueTranslationEventCache = config.lfValueTranslationEventCache
-              .copy(maximumSize = maximumLfValueTranslationCacheEntries),
-            lfValueTranslationContractCache = config.lfValueTranslationContractCache
-              .copy(maximumSize = maximumLfValueTranslationCacheEntries),
-          )
-        ),
+          s"Deprecated! This configuration parameter has no effect. The maximum size of the cache used to deserialize Daml-LF values, in number of allowed entries. By default, nothing is cached."
```

Comment:
@mziolekda Is it okay to communicate deprecation like this? Do we need to make this maybe hidden a well?